saleor/payment/models.py

```diff
@@ -99,14 +99,24 @@ def get_total(self):
     def get_authorized_amount(self):
         money = zero_money()
 
+        # There is no authorized amount anymore when capture is succeeded
+        # since capture can only be made once, even it is a partial capture
+        if self.transactions.filter(
+                kind=TransactionKind.CAPTURE, is_success=True).exists():
+            return money
+
         # Calculate authorized amount from all succeeded auth transactions
         for transaction in self.transactions.filter(
                 kind=TransactionKind.AUTH, is_success=True).all():
             money += Money(
                 transaction.amount, self.currency or settings.DEFAULT_CURRENCY)
 
         # The authorized amount should exclude the already captured amount
-        money -= self.get_captured_amount()
+        for transaction in self.transactions.filter(
```

Comment:
Will this actually run? In 106 you exit the function if there's an existing capture, but here you iterate over them?

Reply:
You catch it. It won't run indeed. It is just put here in case later multiple partial capture is supported though it is unlikely. It shows the correctly logic: total_authorized = authorized - captured. I think it is fine to keep it here though it adds one more db query. I've updated the comment in the code so the developers won't got confused.

Reply:
I changed my mind, I remove this part of code since it effects the test coverage... I've only left the comment in case multiple partial payment is supported later though it is unlikely